src/Object/Contact.php

```diff
@@ -597,7 +597,7 @@ public static function getIdForURL($url, $uid = 0, $no_update = false)
 				'readonly' => 0, 'pending' => 0)
 			);
 
-			$contacts = dba::select('contact', array('id'), array('nurl' => normalise_link($data["url"]), 'uid' => $uid), array('order' => 'id', 'limit' => 2));
+			$contacts = q("SELECT `id` FROM `contact` WHERE `nurl` = '%s' AND `uid` = %d ORDER BY `id` LIMIT 2", dbesc(normalise_link($data["url"])), intval($uid)); 
```

Comment:
What was the problem? Please refer to this comment to get an array of rows from `select()`: https://github.com/friendica/friendica/pull/3933#discussion_r152643959